pulsar-client-go/pulsar/c_go_pulsar.h

```diff
@@ -25,16 +25,6 @@
 
 // Callback proxy functions
 
-void pulsarClientLoggerProxy(pulsar_logger_level_t level, char* file, int line, char* message, void *ctx);
-
-static inline void pulsarClientLoggerConstProxy(pulsar_logger_level_t level, const char* file, int line, const char* message, void *ctx) {
-    pulsarClientLoggerProxy(level, (char*)file, line, (char*)message, ctx);
-}
-
-static inline void _pulsar_client_configuration_set_logger(pulsar_client_configuration_t *conf, void *ctx) {
```

Comment:
This is used to route all the C++ library logs into a Go logger function. If we don't have this, all logs from C++ lib will be printed with a different format on stdout.